src/ItemMeasurer.js

```diff
@@ -0,0 +1,107 @@
+// @flow
+
+import { Component } from 'react';
+import { findDOMNode } from 'react-dom';
+
+import type { Direction } from './createListComponent';
+import type { HandleNewMeasurements } from './DynamicSizeList';
+
+class DOMRectReadOnly {
+  +x: number;
+  +y: number;
+  +width: number;
+  +height: number;
+  +top: number;
+  +right: number;
+  +bottom: number;
+  +left: number;
+}
+
+class ResizeObserverEntry {
+  +target: HTMLElement;
+  +contentRect: DOMRectReadOnly;
+}
+
+type Entries = $ReadOnlyArray<ResizeObserverEntry>;
+
+type ResizeObserverCallback = {
+  // eslint-disable-next-line no-use-before-define
+  (entries: Entries, observer: ResizeObserver): void,
+};
+
+declare class ResizeObserver {
+  constructor(ResizeObserverCallback): ResizeObserver;
+  observe(target: HTMLElement): void;
+  unobserve(target: HTMLElement): void;
+  disconnect(): void;
+}
```

Comment:
Actually flow already has ResizeObserver types :)
https://github.com/facebook/flow/blob/master/lib/bom.js#L446-L456

Reply:
Not the version I'm using (0.80). [`ResizeObserver` defs were added two months ago](https://github.com/facebook/flow/commit/599e26be97f96744b531d416bde47224e70ae573) so I think that means 0.82 or 0.83. I'll update at some point soon.